<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SslTrustPrompt</class>
 <widget class="QDialog" name="SslTrustPrompt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Trust SSL certificate</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QLabel" name="m_MainLabel">
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="m_ContentText"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::No|QDialogButtonBox::Yes</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="2" margin="2"/>
 <resources/>
 <connections/>
</ui>
